docs/resources/rego_policy_instance.md

```diff
@@ -71,15 +71,38 @@ All templates use parameters defined as JSON, below is a list of all the corresp
 
 -> You can also use the Cyral API `GET` `/v1/regopolicies/templates` to retrieve all existing templates and their corresponding parameters schema.
 
-### Data Firewall (data-firewall)
+### Object Protection (object-protection)
```

Comment:
Since we're updating this, maybe add information about the two new templates (ungoverned-statements and fail-closed) should be added as well.